src/notation/view/widgets/editstyle.ui

```diff
@@ -13799,7 +13799,7 @@ By default, they will be placed such as that their right end are at the same lev
                   <item>
                    <widget class="QCheckBox" name="tabParenthArticulation">
                     <property name="text">
-                     <string>Always show parenthesised fret marks when 
+                     <string>Always show parenthesized fret marks when 
 articulation markings are present</string>
```

Comment:
While at it, feel free to remove the line break here. As far as I can see, it serves no purpose.

Reply:
Now I'm on tour, will do tomorrow

Reply:
Hah, works on my tablet